exporting/pubsub/pubsub.c

```diff
@@ -56,6 +56,35 @@ int init_pubsub_instance(struct instance *instance)
     return 0;
 }
 
+/**
+ * Clean a PubSub connector instance
+ *
+ * @param instance an instance data structure.
+ */
+void clean_pubsub_instance(struct instance *instance)
+{
+    info("EXPORTING: cleaning up instance %s ...", instance->config.name);
+
+    struct pubsub_specific_data *connector_specific_data =
+        (struct pubsub_specific_data *)instance->connector_specific_data;
+    pubsub_cleanup(connector_specific_data);
+    freez(connector_specific_data);
+
+    buffer_free(instance->buffer);
```

Comment:
The same thing here, should we test `instance->buffer` before to clean?

Reply:
`buffer_free` checks for NULL values.